toolchain/check/testdata/impl/fail_impl_bad_type.carbon

```diff
@@ -21,7 +21,8 @@ impl true as I {}
 // CHECK:STDOUT:   %.1: type = interface_type @I [template]
 // CHECK:STDOUT:   %Self: %.1 = bind_symbolic_name Self 0 [symbolic]
 // CHECK:STDOUT:   %.2: bool = bool_literal true [template]
-// CHECK:STDOUT:   %.3: <witness> = interface_witness () [template]
+// CHECK:STDOUT:   %.3: type = tuple_type () [template]
```

Comment:
This was added by this PR, but doesn't seem to be referenced by anything and doesn't seem to correspond to anything in the source code. Is there some mistake?

Reply:
I tracked this down -- this is the value representation of the interface type `%.1`. The mapping from complete types to their value representations isn't something we currently display in formatted SemIR. Perhaps it should be.

Reply:
Either that or be given a more informative name?